code/modules/admin/admin_verbs.dm

```diff
@@ -810,14 +810,14 @@ var/list/admin_verbs_mentors = list(
 
 /client/proc/playernotes()
 	set name = "Show Player Info"
-	set category = "Admin"
+	set category = "Admin.Player"
 	if(holder)
 		holder.PlayerNotes()
 	return
 
 /client/proc/free_slot_submap()
 	set name = "Free Job Slot (Submap)"
-	set category = "Admin"
+	set category = "Server"
```

Comment:
How is this server?

Reply:
It's not, however it is redundant since we don't have jobs in submaps (as far as I'm aware thus far)